pages/contributors/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
+import Head from "next/head";
+import ContCard from "components/Contributors/ContCard";
+import Footer from "components/common/Footer";
+import Header from "components/common/Header/Header";
+import styles from "@styles/scss/contributor.module.scss";
+const Contributors = () => {
+  const [contributors, setContributors] = useState([]);
+  useEffect(() => {
+    const fetchContributors = async () => {
+      try {
+        const response = await fetch(
+          "https://api.github.com/repos/aysal04/Project-Amupedia/contributors",
```

Comment:
change this link to the actual repo link, not your fork

Reply:
I will fix it...